src/pineko/scale_variations.py

```diff
@@ -0,0 +1,266 @@
+"""Module to generate scale variations."""
+import pathlib
+from typing import Tuple
+
+import numpy as np
+import pineappl
+import rich
+from eko import beta
+
+from . import check
+
+AS_NORM = 1.0 / (4.0 * np.pi)
+OrderTuple = Tuple[int, int, int, int]
+"""Tuple representing a PineAPPL order."""
+
+
+def sort_qcd_orders(order: OrderTuple) -> int:
+    """Define the sorting for an order. In particular the order is sorted according to the as order."""
+    return order[0]
+
+
+def ren_sv_coeffs(m, max_as, logpart, which_part, nf):
+    """Return the ren_sv contribution relative to the requested log power and perturbative order contribution (which_part).
+
+    Parameters
+    ----------
+    m : int
+        first non zero perturbative order
+    max_as : int
+        max order of alpha_s
+    logpart : int
+        power of the renormalization scale log asked
+    which_part : int
+        asked perturbative order contribution to be rescaled
+    nf : int
+        number of active flavors
+
+    Returns
+    -------
+    float
+        renormalization scale variation contribution
+    """
+    bcoeff = beta.beta_qcd((max_as - logpart - which_part + 2, 0), nf)
+    as_norm_fact = AS_NORM ** (max_as - which_part)
+    m_factor = m
+    if max_as == 0:
+        return 0.0
+    elif max_as == 2:
+        if which_part > 0:
+            m_factor += 1
+        else:
+            if logpart > 1:
+                m_factor = 0.5 * m_factor * (m_factor + 1)
+    return m_factor * as_norm_fact * bcoeff
+
+
+def compute_orders_map(m, max_as, min_al):
```

Comment:
```suggestion
def compute_orders_map(m, max_as, al):
```

No need to call it `min_`, since it is only the order of interest for this function. The fact that is minimum is external to this function, but only relevant for the usage (i.e. for the caller).

Reply:
I'd like a better name than `al`, because it seems the value of $\alpha$, rather than its power.

But this would be part of the orders renaming, also needed for the function above. Let's think about them.

Reply:
[c822719](https://github.com/NNPDF/pineko/pull/75/commits/c822719d67cbef26cc67c418c67a8c44b2876244)

Reply:
I am not marking this as resolved so we can keep in our mind the naming business